website/docs/r/client_vpn_endpoint.html.markdown

```diff
@@ -0,0 +1,76 @@
+---
+layout: "aws"
+page_title: "AWS: aws_client_vpn_endpoint"
+sidebar_current: "docs-aws-resource-client-vpn-endpoint"
+description: |-
+  Provides an AWS Client VPN endpoint for OpenVPN clients.
+---
+
+# aws_client_vpn_endpoint
+
+Provides an AWS Client VPN endpoint for OpenVPN clients. For more information on usage, please see the 
+[AWS Client VPN Administrator's Guide](https://docs.aws.amazon.com/vpn/latest/clientvpn-admin/what-is.html).
+
+## Example Usage
+
+```hcl
+resource "aws_client_vpn_endpoint" "example" {
+  description = "terraform-clientvpn-example"
+  server_certificate_arn = "${aws_acm_certificate.cert.arn}"
+  client_cidr_block = "10.0.0.0/16"
+
+  authentication_options {
+    type = "certificate-authentication"
+    root_certificate_chain_arn = "${aws_acm_certificate.root_cert.arn}"
+  }
+
+  connection_log_options {
+	  enabled = true
```

Comment:
Nit: Could reformat the example (too many spaces here)

Reply:
Done